pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -446,13 +446,29 @@ func configureQemuConf(qemuFilename string) (err error) {
 		return err
 	}
 
-	if envVarValue, ok := os.LookupEnv("VIRTIOFSD_DEBUG_LOGS"); ok && (envVarValue == "1") {
+	if debugLogsStr, ok := os.LookupEnv("VIRTIOFSD_DEBUG_LOGS"); ok && (debugLogsStr == "1") {
 		_, err = qemuConf.WriteString("virtiofsd_debug = 1\n")
 		if err != nil {
 			return err
 		}
 	}
 
+	if pathsStr, ok := os.LookupEnv(services.ENV_VAR_SHARED_FILESYSTEM_PATHS); ok {
```

Comment:
@akalenyu can we add a unit test which verifies the qemu config?

Reply:
tbh I thought about it but I don't think we have a framework in place to test parts of `pkg/virt-launcher/virtwrap/util/libvirt_helper.go`. Should I try to come up with one? Do you have any suggestions?

Reply:
What to simply pass a temporary file to configureQemuConf and simply check its content?

Reply:
oh, yes, can do that